plugin/pkg/scheduler/api/v1/types.go

```diff
@@ -128,24 +128,36 @@ type ExtenderConfig struct {
 	// HTTPTimeout specifies the timeout duration for a call to the extender. Filter timeout fails the scheduling of the pod. Prioritize
 	// timeout is ignored, k8s/other extenders priorities are used to select the node.
 	HTTPTimeout time.Duration `json:"httpTimeout,omitempty"`
+	// NodeCacheCapable specifies that the extender is capable of caching node information,
```

Comment:
@timothysc @davidopp @ravigadde 
I fixed all the comments as suggested and pushed the changes. Also, added omitempty tag for node list in ExtenderArgs. When the node list is empty, older implementations will just see empty list of nodes and they should be able to handle that.